client/src/components/Profile/Profile.js

```diff
@@ -1,8 +1,39 @@
-import React from "react";
+import { React, useState } from "react";
 import PropTypes from "prop-types";
+import * as fetch from "../../lib/fetch";
```

Comment:
```suggestion
import * as fetcher from "../../lib/fetcher";
```

This avoids confusion with the built-in `fetch()` function.